### ISSUE TITLE
Set Default to L/D

### ISSUE THREAD

**Issue by Kerbinator-Fras** [DBooots/KerbalWindTunnel#8]
When setting default to flight envelope, this mod will crash KSP even with 2-part vessel
Now to avoid this I mainly "feed" KWT with a single-part vessel and then when its envelope calculated to 100%, I switch to L/D (at default 100m/s) and then use my ship
in fact, for design of extremely-optimized stock vessels, L/D is the most useful, and relatively easier on CPU.

**Comment by DBooots** [comment]
Can you expand a little more on what you're hoping for? Do you mean that the default graph for the flight envelope graph selection should be L/D?
The crashes should be fixed in the next release.

**Comment by Kerbinator-Fras** [comment]
Yes you are exactly right.
L/D is the least laggy display, and is very important for optimizations
(both velcurve and envelope is far more likely to lag or crash your KSP)

**Comment by DBooots** [comment]
There's a new pre-release out that will hopefully fix the lag and crashes. I encourage you to try it and let me know your experience(https://github.com/DBooots/KerbalWindTunnel/releases/tag/1.2.5.-1). I've switched to using new .Net functionality to make it much more robust and reliable. And I've added better progress reporting as it calculates.
Based on your answer, I think my initial understanding of your request is not quite right: you'd like the AoA-based graphs to be the default?
